tests/network/hotplug.go

```diff
@@ -411,3 +427,12 @@ func cleanMACAddressesFromSpec(status []v1.Interface) []v1.Interface {
 	}
 	return status
 }
+
+func migrate(vmi *v1.VirtualMachineInstance) {
+	checks.SkipIfMigrationIsNotPossible()
```

Comment:
Please remove this skip. Having the label should be enough.

Reply:
The label is here https://github.com/kubevirt/kubevirt/pull/9468/files#diff-2be6ff8bee7a7b41764cd824e3e7afbc1ae3facd2893bdeb60f7ac978303a387R41

So now it can happen that I specify this label(meaning I **really** want to run these tests) but I did a mistake in my setup. Therefore this will be "silently" skipped. Bummer.

Reply:
OK so by means of having the labels should be enough you don't mean that labels should make the cluster adaptive to the test requirements, you actually mean that you want the tests to fail in purpose, because you want to be notified about the mistake done with the setup.

Reply:
> So now it can happen that I specify this label(meaning I really want to run these tests) but I did a mistake in my setup. Therefore this will be "silently" skipped. Bummer.

I will add `Expect(checks.HasFeature(virtconfig.LiveMigrationGate)).To(BeTrue())` to the migration based tests instead.

Reply:
>@xpivarc in the sig-compute parallel migration lanes, are we failing if there are not enough nodes for migration, or skipping?

I believe we are skipping but I would need to check. Let me just point out it does not matter, this is a afford that I am trying to push, and starting with new code(PRs) is the best place to start.

Reply:
@xpivarc I got your idea, I will adopt it as well since it makes sense to me, already applied it at #9470

Reply:
done.